### PR TITLE
Always clear undo stack after a mutation on any panel

### PR DESCRIPTION
**Pull request by rwols**
This prevents memory growing infinitely due to Sublime Text maintaining an
infinite undo stack, even for panels.

### CODE REVIEW

File: plugin/core/panels.py [hunk 1]
```diff
@@ -37,9 +37,19 @@ class PanelName:
 
 @contextmanager
 def mutable(view: sublime.View) -> Generator:
+    # Unset the read-only flag temporarily
     view.set_read_only(False)
-    yield
-    view.set_read_only(True)
+    try:
+        yield
+    finally:
```
Comment (rchl) [inline plugin/core/panels.py:44]: Why within an exception handler now? Do you of any case when this could happen?

**Review by rchl — APPROVED** [review]

File: plugin/core/panels.py (hunk 1 above)
Comment (rchl) [reply plugin/core/panels.py:44]: yes but I'm not sure why would we get exceptions triggered within "with" and whether we want to hide them like that

Comment (rwols) [reply plugin/core/panels.py:44]: the exception is not consumed, only the `finally` block is executed to do cleanup and the exception will then keep going up the call stack.

**Comment by jfcherng** [comment]
https://github.com/sublimelsp/LSP/blob/c3075e979b3ced89ba2a4a4417f1f101157eaaf2/plugin/core/panels.py#L45-L50
Unfortunately, the console says `Warning: Can't clear undo stack from text command`.

**Pull request merged**